test/fastboot-response-test.js

```diff
@@ -0,0 +1,52 @@
+var expect = require('chai').expect;
+var path = require('path');
+var FastBootHeaders = require('../lib/fastboot-headers.js');
+var FastBootResponse = require('../lib/fastboot-response.js');
+
+describe("FastBootResponse", function() {
+  var fastBootResponse;
+
+  beforeEach(function () {
+    var mockResponse = {
+      headers: {
+        "i-am-a": "mock header, me too"
+      },
+      get: function () {
+        return this.headers.cookie;
+      }
+    }
+
+    fastBootResponse = new FastBootResponse(mockResponse);
+  });
+
+  describe("attributes", function () {
+    var expectedProperties = [
+      "headers",
+      "statusCode"
+    ];
+
+    expectedProperties.forEach(function (property) {
```

Comment:
We should assert the 2 properties, instead of generating tests for the properties on the fly